.changeset/lazy-roses-drive.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/preview-middleware': patch
+'@sap-ux/ui5-config': patch
```

Comment:
We do not need to change `ui5-config`. Revert any any changes to `ui5-config` and remove this please.